src/libs/autoScrollback/index.web.js

```diff
@@ -0,0 +1,56 @@
+/* autoScrollBack address Mobile Safari-specific issues when the user overscrolls the window while the keyboard is visible */
+import Str from 'expensify-common/lib/str';
+
+import getBrowser from '../getBrowser';
+
+const init = () => {
+    if (getBrowser() !== 'safari') {
+        return;
+    }
+    const userAgent = navigator.userAgent.toLowerCase();
+
+    if (Str.contains(userAgent, 'iphone os 1')) {
+        const innterHeightWithoutKeyboard = window.innerHeight;
+
+        let waitingForScrollAfterFocus = false;
+
```

Comment:
This line is not helping

Reply:
Can you elaborate?

Reply:
I mean It's not making it more readable. I feel removing this is better or adding comments to explain these variables